Scripts/Prerequisite.ps1

```diff
@@ -70,7 +71,17 @@ try {
 
   Write-Host "Creating selfsigned certificate in keyvault ..." -ForegroundColor White
   az keyvault certificate create --vault-name $keyVaultName -n $certName  --policy `@PEMCertCreationPolicy.json
```

Comment:
is it required to use PEMCertCreationPolicy while creating the certificate ? this was one of the option we tried to get appropriate content while downloading the cert, now that we have switched to secret, please invalidate if its really required.

Reply:
@saurabh-mridul - Yes, We needed PEMCertCreationPolicy. There are two thing specific to cert

1. Create cert
2. Download Cert
TO create PEM cert we need this policy earlier we were using default policy which creates pkcs12 type so we added this json to add PEM cert. downloading certificate was downloading public cert not the private key so used download secret command to download both private and public cert as part of PEM file.